weed_notification: '/weed_found'

row_end_y: 6.5
row_start_y: -6.5
row_location_x: [-3.75,-2.75,0.75,0.25,2.25,3.25]



